packages/connect/src/api/getAddress.ts

```diff
@@ -30,6 +30,14 @@ export default class GetAddress extends AbstractMethod<'getAddress', Params[]> {
             ? { ...this.payload, bundle: [this.payload] }
             : this.payload;
 
+        // Workaround to allow empty signature in multisig (issue #10841)
+        payload?.bundle.forEach(bundleElement => {
+            if (bundleElement.multisig && bundleElement.multisig?.signatures === undefined) {
+                bundleElement.multisig.signatures = Array(
```

Comment:
cc @martykan shoudln't we put some console warn here?

Reply:
I was thinking that we could make this parameter officially optional if it's not used in GetAddress, overall the multisig param needs proper docs and could refactoring (for example it supports two variants of passing pubkeys which are mutually exclusive, but the type doesn't check that.

Reply:
But it is currently generated from Protobuf so I'm not sure at what level to handle it